examples/specs/sequence_line_fold.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
+  "description": "Plots two functions using a generated sequence.",
+  "width": 300,
+  "height": 150,
+  "data": {
+    "sequence": {
+      "start": 0,
+      "stop": 12.7,
+      "step": 0.1,
+      "as": "x"
+    }
+  },
+  "transform": [
+    {
+      "calculate": "sin(datum.x)",
+      "as": "sin(x)"
+    },
+    {
+      "calculate": "cos(datum.x)",
+      "as": "cos(x)"
+    },
+    {
+      "fold": ["sin(x)", "cos(x)"]
```

Comment:
It's worth noting that folding means we can't produce tooltip for both lines at the same time.  We should think about repeat layer with scale for each layer so we can make a legend.  
(Just noting, I think this PR is fine.)

Reply:
I'm okay with that limitation.

Reply:
FWIW, I can tell that in practice that is too limiting for adoption. 
People wanna compare different lines all the time. Let's discuss more in https://github.com/vega/vega-lite/issues/1274.